app.config.js

```diff
@@ -256,7 +251,7 @@ module.exports = function (config) {
         './plugins/starterPackAppClipExtension/withStarterPackAppClip.js',
         './plugins/withAndroidManifestPlugin.js',
         './plugins/withAndroidManifestFCMIconPlugin.js',
-        './plugins/withAndroidStylesWindowBackgroundPlugin.js',
+        // './plugins/withAndroidStylesWindowBackgroundPlugin.js',
```

Comment:
this was added in https://github.com/bluesky-social/social-app/pull/3105. is it no longer needed?

Reply:
from my understanding, this is now handled by the `expo-splash-screen` config plugin